index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap' rel='stylesheet' />
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Portfolio</title>
+
+</head>
+
+<body>
+    <nav>
+        <div class="appbar-compact">
+            <div class="toolbar">
+                <div class="logo-placeholder">
+                    <a class="navbar-brand" href="#">My Logo</a>
+                </div>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </div>
+        </div>
+    </nav>
+
+    <header>
+        <div class="headline">
+            <p class="primary-text1">I’m Fatuma Glad to see you!</p>
+            <p class="primary-text2">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="socialmedia">
+                <div class="supporting-text">
+                    <p class="connect">LET’S CONNECT</p>
+                </div>
+            </div>
+
+            <div class="socialmedia-icons">
+                <img class="twitter" src="images/twitter.png">
+                <img class="linkedin" src="images/Linkedin icon.png">
+                <img class="m" src="images/m.png">
+                <img class="github" src="images/github.png">
+                <img class="vector" src="images/Vector.png">
+            </div>
```

Comment:
- [ ] For proper semantics, the list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
- Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

Reply:
👍